package.json

```
@@ -115,18 +115,19 @@
     "react-i18next": "11.18.6",
     "react-icons": "4.11.0",
     "react-lottie": "1.2.3",
+    "react-markdown": "^10.1.0",
     "react-masonry-css": "1.0.16",
     "react-modal": "3.14.4",
     "react-router-dom": "5.3.0",
     "react-timer-hook": "3.0.5",
     "react-transition-group": "4.4.2",
     "react-virtualized": "9.22.3",
     "reflect-metadata": "0.1.13",
+    "remark-gfm": "^4.0.1",
```

please pin this version
```suggestion
    "remark-gfm": "4.0.1",
```